build-system/babel-plugins/babel-plugin-transform-minified-comments/index.js

```diff
@@ -14,28 +14,32 @@
  * limitations under the License.
  */
 
+/**
+ * Shortens a comment block by removing line breaks and consecutive whitespaces.
+ * @param {Array<Object>} comments
+ * @return {Array<Object>}
+ */
+function formatComments(comments) {
+  return comments.map((comment) => ({
+    ...comment,
+    value: comment.value.replace(/\n/g, ' ').replace(/\s+/g, ' '),
+  }));
+}
+
 // Ensure comments in minified build output is minimal.
 module.exports = function () {
   return {
     visitor: {
       Statement(path) {
         const {node} = path;
         const {trailingComments} = node;
-        if (!trailingComments || trailingComments.length <= 0) {
-          return;
+        if (trailingComments?.length) {
+          node.trailingComments = formatComments(trailingComments);
```

Comment:
Nit: I'd likely just inline this, but the comment on the JSDoc is nice.

Reply:
Agree, I've inlined it since there's just one call-site.